packages/immutable/test/index.test.ts

```diff
@@ -11,14 +11,15 @@ import {
 } from "../src";
 import { suite } from "uvu";
 import * as assert from "uvu/assert";
-import { cloneDeep } from "lodash";
+
+const cloneDeep = <T>(obj: T): T => JSON.parse(JSON.stringify(obj));
```

Comment:
Yeah I saw that post too :P
It's not on node though, at least not in 16
Will try to go back to lodash I think...

Reply:
FWIW, it's in Node 17+. Time to get on Node 18. 😄 

(Thanks for the heads up on structuredClone, though; I didn't know it existed!)

Reply:
There should be a polyfill available; you can use the same pattern that I used in fetch/src/server.ts to make sure it's an optional dependency.

Reply:
Side fact: I did a proposal for Object.clone() at the same time, but they took the less powerful proposal from the Mozilla guy.

Reply:
Ok, this is only for testing, and the structuredClone doesn't allow cloning functions, which were the part of the initial tests, but don't have to be. `JSON.parse(JSON.stringify(obj))` works just fine otherwise, and it doesn't require a polyfill just to be able to test the api.
But yeah, if we need to clone-deep something as a part of the api, `structuredClone` would be a go-to function.